services-manager/src/main/resources/application-example-model.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+spring:
+  profiles:
+    include:
+      - example-libs
+      - example-model-config

```

Comment:
While example-model is the module I'm not sure that this is the best name for the profile. Can we make it a more descriptive name? Especially as the module name might be changed in the future.

Reply:
Pending rename of the module.
For now it makes most sense that it matches the module name, but should we rename to a more-appropriate example-service or similar I could see this changing to match.